CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 10.0.2
```

Comment:
Aren't these version numbers for npm releases?

Reply:
Happy to remove this, thanks!

Reply:
To be clear, I'm not sure. Let's wait for @stephaniehobson to weigh-in.

Reply:
Just to confirm, these are indeed versioned releases for the Protocol npm package. This sort of change doesn't need to be reflected in the changelog.